test/models/compute_resources/ovirt_test.rb

```diff
@@ -13,6 +13,18 @@ class OvirtTest < ActiveSupport::TestCase
     assert_equal host, as_admin { cr.associated_host(vm) }
   end
 
+  describe 'validate' do
+    test 'valid if url is present' do
```

Comment:
Can we use shoulda matchers to check this to reduce boilerplate?
Please also add a test for the URL regex. Shoulda matchers should be fine.

```ruby
should allow_values('http://foo.com', 'http://bar.com/baz').
    for(:url)

should_not allow_values('http://foo.com', 'buz').
    for(:url)
```

In addition, i think it's safe to add `https://github.com/theforeman/foreman/blob/develop/app/validators/url_schema_validator.rb` and limit the allowed schemes to http and https.

```
validates :url, :url_schema => ['http', 'https']
```

Please add tests (shoulda matchers) for this, too. Positive and Negative should be fine.